roles/kubernetes-apps/metallb/templates/metallb.yml.j2

```diff
@@ -345,11 +345,19 @@ spec:
         kubernetes.io/os: linux
       serviceAccountName: speaker
       terminationGracePeriodSeconds: 2
+{% if metallb_speaker_nodeselector %}
+      nodeSelector:
+        {{ metallb_speaker_nodeselector | to_nice_yaml | indent(width=8) }}
+{%- endif %}
+{% if metallb_speaker_tolerations %}
       tolerations:
-      - effect: NoSchedule
-        key: node-role.kubernetes.io/master
-      - effect: NoSchedule
-        key: node-role.kubernetes.io/control-plane
+        {{ metallb_speaker_tolerations | to_nice_yaml(indent=2) | indent(width=8) }}
+{% endif %}
+      # tolerations:
+      # - effect: NoSchedule
+      #   key: node-role.kubernetes.io/master
+      # - effect: NoSchedule
+      #   key: node-role.kubernetes.io/control-plane
```

Comment:
Why do we need to add commented-out lines 356-360?
Or did you want to keep the same lines for else case instead of commented-out lines?

Reply:
My bad, this was for testing purposes before creating the variables. I'll remove it!